app/assets/javascripts/fae/vendor/jquery.multi-select.js

```diff
@@ -221,10 +221,12 @@
               tabindex = (e.shiftKey) ? tabindex-1 : tabindex+1;
               $('[tabindex="'+(tabindex)+'"]').focus();
               return;
-            }else{
-              if(e.shiftKey){
-                that.$element.trigger('focus');
-              }
+            } else {
```

Comment:
I'm not a big fan of hacking libraries like this. If the time comes to upgrade, this type of thing could easily get lost in the mix. However, I don't have a simple alternative.